app/app_config.go

```diff
@@ -81,6 +89,10 @@ var (
 		stakingtypes.ModuleName,
 		authz.ModuleName,
 		genutiltypes.ModuleName,
+
+		// IBC
+		ibcexported.ModuleName,
+		ibctransfertypes.ModuleName,
```

Comment:
transfer does not have a BeginBlocker lifecycle hook, but I think it would just noop if included here, but this is unnecessary

Reply:
These were listed in the IBC integration guide: https://ibc.cosmos.network/v10/ibc/integration/#application-abci-ordering

Do you know if they are there "just in case" for future proofing, to ensure automatic wiring if need arises, or if those will guaranteed never be needed and hence would make sense to be dropped?